engine/runtime/src/main/java/org/enso/interpreter/node/callable/InvokeCallableNode.java

```diff
@@ -375,7 +385,16 @@ static Object doPolyglot(
   @Fallback
   public Object invokeGeneric(
       Object callable, VirtualFrame callerFrame, State state, Object[] arguments) {
-    Atom error = EnsoContext.get(this).getBuiltins().error().makeNotInvokable(callable);
+    Atom cause = null;
```

Comment:
The same code is already present in `IndirectInvokeCallableNode` once. Can we refactor it to a single method?

Reply:
Hmmm, perhaps a good point.

Any suggestions where this common function could live? Put it as `static` in `InvokeCallableNode` and call from `IndirectInvokeCallableNode`?

Reply:
Done in [97b7d27](https://github.com/enso-org/enso/pull/12238/commits/97b7d274adecd13f52f64b41fde44b54ccd0c1de)